src/main/java/nl/asd/workplace/domain/Building.java

```diff
@@ -16,13 +19,59 @@
     private HashMap<DayOfWeek, OpeningTime> openingHours;
     private List<Workplace> workplaces;
 
+    public Building(BuildingId id, String name) {
+        this.id = id;
+        this.name = name;
+        this.openingHours = standardOpeningHours();
+        this.workplaces = new ArrayList<>();
+    }
+
     public Building(BuildingId id, String name, HashMap<DayOfWeek, OpeningTime> openingHours) {
         this.id = id;
         this.name = name;
         this.openingHours = openingHours;
         this.workplaces = new ArrayList<>();
     }
 
+    /**
+     * @return HashMap with days: Mo-Fr | OpeningHours: 08:00-18:00
+     */
+    private HashMap<DayOfWeek, OpeningTime> standardOpeningHours() {
+        return Stream.of(
+                DayOfWeek.MONDAY, DayOfWeek.TUESDAY, DayOfWeek.WEDNESDAY, DayOfWeek.THURSDAY, DayOfWeek.FRIDAY, DayOfWeek.SATURDAY, DayOfWeek.SUNDAY)
+                .collect(Collectors.toMap(
+                        dayOfWeek ->
+                                dayOfWeek, dayOfWeek ->
+                                new OpeningTime(LocalTime.of(8, 0), LocalTime.of(18, 0)), (a, b) -> b, HashMap::new));
+    }
```

Comment:
I don't really agree with this, what if you want to change the opening hours of a building... it's impossible to do now

You are also over complicating things with these streams (it doesn't increase readability)
```suggestion
    private HashMap<DayOfWeek, OpeningTime> standardOpeningHours() {
        var map = new HashMap<DayOfWeek, OpeningTime>();
        var standardOpeningTime = new OpeningTime(LocalTime.of(8, 0)
        for (var day : List.of(DayOfWeek.MONDAY, DayOfWeek.TUESDAY, DayOfWeek.WEDNESDAY, DayOfWeek.THURSDAY, DayOfWeek.FRIDAY, DayOfWeek.SATURDAY, DayOfWeek.SUNDAY) {
            map.put(day, standardOpeningTime);
        }
        
        return map;
    }
```
Is probably what I would do instead of this (it probably is faster, too), but we **want** to change the openinghours